lang/en/attendance.php

```diff
@@ -244,6 +244,9 @@
 $string['includeall'] = 'Select all sessions';
 $string['includenottaken'] = 'Include not taken sessions';
 $string['includeqrcode'] = 'Include QR code';
+$string['rotateqrcode'] = 'Rotate QR code';
+$string['rotateqrcodeinterval'] = 'Rotate QR code interval';
+$string['rotateqrcodeinterval_desc'] = 'Time interval to rotate QR codes by.';
```

Comment:
It is not clear what an "interval" is - it might be better to replace this with something like "Rotate QR code interval (seconds)" or "The time in seconds to rotate the QR code."